### PR TITLE
feat: require LXD 5.0 for the appliance

### PR DESCRIPTION
**Pull request by morphis**
Aligns with what the appliance will start to enforce upon installation.

### CODE REVIEW

File: howto/install-appliance/finalise.md [hunk 1]
```diff
@@ -1,6 +1,20 @@
 After installing the Anbox Cloud Appliance, you must initialise it and register your Ubuntu SSO account with the dashboard.
 
 <a name="initialise"></a>
```
Comment (ru-fu) [inline howto/install-appliance/finalise.md:3]: The anchor should stay with the "Initialise" title below.

File: howto/install-appliance/finalise.md [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+## Install requirements
+
+LXD >= 5.0 is required for the Anbox Cloud Appliance. Check which version you have currently installed:
```
Comment (ru-fu) [inline howto/install-appliance/finalise.md:6]:
```suggestion
The Anbox Cloud Appliance requires LXD >=5.0. Check which version you have currently installed:
```

File: howto/install-appliance/finalise.md [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+
+    lxd --version
+
+If LXD is not installed, run
```
Comment (ru-fu) [inline howto/install-appliance/finalise.md:10]:
```suggestion
If LXD is not installed, run:
```

File: howto/install-appliance/finalise.md [hunk 4]
```diff
@@ -1,6 +1,20 @@
...
+
+    snap install --channel=5.0/stable lxd
+
+If LXD is already installed but the version is older than 5.0, run
```
Comment (ru-fu) [inline howto/install-appliance/finalise.md:14]:
```suggestion
If LXD is already installed but the version is older than 5.0, run:
```

File: requirements.md [hunk 5]
```diff
@@ -56,6 +56,13 @@ The Anbox Cloud Appliance supports the following Ubuntu versions:
 * 20.04 (focal)
 * 22.04 (jammy)
 
+<a name="appliance-lxd-version"></a>
+### LXD version
+
+The Anbox Cloud appliance currently supports the following LXD versions
```
Comment (ru-fu) [inline requirements.md:62]:
```suggestion
The Anbox Cloud Appliance currently supports the following LXD versions:
```

File: requirements.md [hunk 6]
```diff
@@ -71,6 +78,13 @@ Anbox Cloud supports the following Ubuntu versions:
 
 For new deployments, Ubuntu 20.04 (focal) is preferred.
 
+<a name="juju-lxd-version"></a>
+### LXD version
+
+Anbox Cloud currently supports the following LXD versions
```
Comment (ru-fu) [inline requirements.md:84]:
```suggestion
Anbox Cloud currently supports the following LXD versions:
```

File: tut/installing-appliance.md [hunk 7]
```diff
@@ -35,15 +35,29 @@ Run the following commands to ensure that all installed packages on your system
     sudo apt update
     sudo apt upgrade
 
-### 2. Attach your machine to the UA subscription
+### 2. Install dependencies
+
+LXD >= 5.0 is required for the Anbox Cloud Appliance. Check which version you have currently installed:
```
Comment (ru-fu) [inline tut/installing-appliance.md:40]:
```suggestion
The Anbox Cloud Appliance requires LXD >= 5.0. Check which version you have currently installed:
```

File: tut/installing-appliance.md [hunk 8]
```diff
@@ -35,15 +35,29 @@ Run the following commands to ensure that all installed packages on your system
...
+
+    lxd --version
+
+If LXD is not installed, run
```
Comment (ru-fu) [inline tut/installing-appliance.md:44]:
```suggestion
If LXD is not installed, run:
```

File: tut/installing-appliance.md [hunk 9]
```diff
@@ -35,15 +35,29 @@ Run the following commands to ensure that all installed packages on your system
...
+
+    snap install --channel=5.0/stable lxd
+
+If LXD is already installed but the version is older than 5.0, run
```
Comment (ru-fu) [inline tut/installing-appliance.md:48]:
```suggestion
If LXD is already installed but the version is older than 5.0, run:
```

File: tut/installing-appliance.md [hunk 10]
```diff
@@ -35,15 +35,29 @@ Run the following commands to ensure that all installed packages on your system
...
+### 3. Attach your machine to the UA subscription
 
 The Anbox Cloud Appliance requires a valid Ubuntu Advantage for Applications subscription.
 
 Before installing the appliance, you must attach the machine on which you want to run the Anbox Cloud Appliance to your Ubuntu Advantage for Applications subscription. To do so, run the following command, replacing *<UA_token>* with your Ubuntu Advantage for Applications token:
 
     sudo ua attach <UA_token>
 
-### 3. Install the snap
+### 4. Install the snap
```
Comment (ru-fu) [inline tut/installing-appliance.md:60]: There's a "4. Install additional tools" underneath that must be "5." now.

File: howto/install-appliance/finalise.md (hunk 1 above)
Comment (morphis) [reply howto/install-appliance/finalise.md:3]: Fixed

File: tut/installing-appliance.md (hunk 10 above)
Comment (morphis) [reply tut/installing-appliance.md:60]: Fixed

**Review by ru-fu — APPROVED** [review]
Looks good!

**Pull request merged**